es-ds-components/components/es-slider.vue

```diff
@@ -126,7 +129,7 @@ model.value = props.startingValue;
     bottom: 27px;
     box-shadow: 0 1px 6px 0 rgba(34, 38, 51, 0.25);
     color: variables.$white;
-    content: attr(aria-valuenow);
+    content: v-bind("`'${model}'`");
```

Comment:
The syntax is super-weird because it's evaluating it the same way it would an attribute in the HTML section. But basically the CSS needs `'string content'`, and with *setup* you need to add `' '` (or `" "`) to the *v-bind*.